utils/smstools3/files/smstools3.init

```diff
@@ -1,43 +1,22 @@
 #!/bin/sh /etc/rc.common
-# Copyright (C) 2014 OpenWrt.org
-# smsd initscript openwrt mod
+# Copyright (C) 2014-2018 OpenWrt.org
 
 START=99
```

Comment:
highest start level should usually be < 95 (S95 called the "boot done" script

Reply:
I wondered about that too, but thought that maybe it is intentional, so didn't dare change it. (Maybe nothing ever should depend on this service because it is a tad unreliable?)

But I'm happy to change it to 94, if this is more correct.